sklearn/linear_model/_least_angle.py

```diff
@@ -1373,8 +1374,9 @@ class LarsCV(Lars):
 
     Attributes
     ----------
-    active_ : list of length n_alphas or list of thereof of shape (n_targets,)
+    active_ : list of length n_alphas or list thereof
```

Comment:
I also find this hard to understand (too implicit). The following would be easier to understand (assuming it fits within 80 columns):

```suggestion
    active_ : list of length n_alphas or list of such lists
```